### PR TITLE
Clear method api [HZAI-32]

### PR DESCRIPTION
**Pull request by shultseva**
Clear vector collection method api

### CODE REVIEW

**Review by k-jamroz — APPROVED** [review]

File: protocol-definitions/VectorCollection.yaml [hunk 1]
```diff
@@ -272,3 +272,19 @@ methods:
             A null value triggers the optimization of the only index within the collection.
     response: {}
 
+  - id: 10
+    name: clear
+    since: 2.8
+    doc: |
+      Clear vector collection.
+    request:
+      retryable: true
```
Comment (k-jamroz) [inline protocol-definitions/VectorCollection.yaml:281]: I wonder if it should be retryable. IMap.clear is not

Comment (vbekiaris) [reply protocol-definitions/VectorCollection.yaml:281]: As a mutating operation, it's safer to go with `false`

**Review by vbekiaris — CHANGES_REQUESTED** [review]
@shultseva I think `retryable` value should be `false`

File: protocol-definitions/VectorCollection.yaml (hunk 1 above)
Comment (shultseva) [reply protocol-definitions/VectorCollection.yaml]: 03064e0df406a2e64f6b378874f28fece5a9da18

**Review by vbekiaris — APPROVED** [review]

**Pull request merged**